### PR TITLE
tw-rs: init at 0.1.26

### PR DESCRIPTION
**Pull request by vmchale**
###### Motivation for this change
Add [tw-rs](https://github.com/vmchale/tw-rs) to nixpkgs

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/misc/tw-rs/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -p target/release/tw $out/bin/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Twitter command-line interface written in rust";
+    homepage = https://github.com/vmchale/tw-rs;
+    license = licenses.bsd3;
+    #maintainers = with maintainers; [ gebner ];
```
Comment (Mic92) [inline pkgs/misc/tw-rs/default.nix]: Since this is your project, do you want to maintain this?

File: pkgs/misc/tw-rs/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "1s1gk2wcs3792gdzrngksczz3gma5kv02ni2jqrhib8l6z8mg9ia";
+    };
+  buildInputs = [ perl zlib openssl ];
+
+  depsSha256 = "1lg1jh6f9w28i94vaj62r859g6raalxmxabvw7av6sqr0hr56p05";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -p target/release/tw $out/bin/
+  '';
```
Comment (Mic92) [inline pkgs/misc/tw-rs/default.nix]: I was able to remove this phase, when building the package

**Comment by Mic92** [comment]
I squashed your commits into one. It would be helpful in the future to not use `git pull` in a pull request, as it creates a merge commit. Instead a `git pull --rebase` or `git fetch origin master` (if `origin` points to `nixos/nixpkgs`) followed by `git rebase origin/master` makes it easier for maintainer.

**Comment by vcunat** [comment]
Actually added you to the maintainer list in 477fa42be6.

**Pull request closed without merging**